core/src/main/scala/BiggerApplies.scala

```diff
@@ -1,34 +0,0 @@
-//: ----------------------------------------------------------------------------
-//: Copyright (C) 2017 Verizon.  All Rights Reserved.
-//:
-//:   Licensed under the Apache License, Version 2.0 (the "License");
-//:   you may not use this file except in compliance with the License.
-//:   You may obtain a copy of the License at
-//:
-//:       http://www.apache.org/licenses/LICENSE-2.0
-//:
-//:   Unless required by applicable law or agreed to in writing, software
-//:   distributed under the License is distributed on an "AS IS" BASIS,
-//:   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//:   See the License for the specific language governing permissions and
-//:   limitations under the License.
-//:
-//: ----------------------------------------------------------------------------
-package nelson
-
-import scalaz._
-
-// Triskaidekaphobia
-object BiggerApplies {
```

Comment:
It was a sad day when I had to write this (or was that Ross, I can't remember), but glad it see it go. Assuming you are using `mapN` in places where this would have been applicable.

Reply:
Yessir, I always wondered if they bothered to go up to 13, why not just go up the rest of the way to 22?